brave/brave-core
# Adjust publisher verified checkmark on the tip banner

Resolves https://github.com/brave/brave-browser/issues/12363

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.

## Test Plan:

- Start browser with a clean profile.
- Visit an unverified publisher.
- Open the rewards panel and click "Enable Rewards".
- Click "Send a tip".
  - Verify that no "X" mark is displayed next to their icon.
- Visit a verified publisher.
- Open the rewards panel and click "Enable Rewards".
- Click "Send a tip".
  - Verify that a checkmark is displayed next to their icon, matching the image [here](https://github.com/brave/brave-browser/issues/12363#issuecomment-731244582).
